browser/ui/browser_commands.cc

```diff
@@ -6,6 +6,8 @@
 #include "brave/browser/ui/browser_commands.h"
 
 #include "base/files/file_path.h"
+#include "brave/browser/ui/views/frame/brave_browser_view.h"
```

Comment:
this needs to be in `#if defined(TOOLKIT_VIEWS)` to match gn config - the gn config is possibly not relevant since all of this is `!is_android`, but we should make sure they match in any case

Reply:
Fixed in 69f8b573f8508373220d3f1f4ae9578ac065d6c9